File: src/main/java/com/app/pojos/NewRequest.java
package com.app.pojos;

import java.util.List;

import javax.persistence.CascadeType;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.OneToMany;
import javax.persistence.Table;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@Entity
@Table(name="NewRequestTable")
public class NewRequest {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Integer request_id;

	private String user_id;
	
	@Column(length = 30)
	private String category;
	
	@Column(length = 30)
	private String dimension;
	
	private double weight;
	
	@Column(length = 30 )
	private String controller; // arduino esp32
	
	@Column(length = 30)
	private String sensor; // 
	
	@Column(length = 30)
	private String build_type;
	
	@Column(length = 30)
	private String motor_type; // stepper or geared
	
	@Column(length = 30)
	private String wiring_type;
	
	@Column(length = 30)
	private String power_supply_unit; // smps transformer
	
	@Column(length = 30)
	private String bluetooth_enabled; // if yes then type 

	private boolean internet_controlled; // yes no only
	
	@Column(length = 30)
	private String camera_equipped; // if yes then specify
	
	@Column(length = 30)
	private String physical_remote; // if yes then specify

	@Column(length = 500)
	private String working;
	
}

File: MySmartSpace_DOTNET/BOL/NewRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BOL
{
    public class NewRequest
    {
        public int RequestId { get; set; }
        public User UserId { get; set; }
        public string RequestDate { get; set; }
        public ProductCategory Category { get; set; }
        public string Dimension { get; set; }
        public double Weight { get; set; }
        public string Controller { get; set; }
        public string Sensor { get; set; }
        public string BuildType { get; set; }
        public string MotorType { get; set; }
        public string WiringType { get; set; }
        public string PowerSupplyUnit { get; set; }
        public string BluetoothEnabled { get; set; }
        public bool InternetControlled { get; set; }
        public string CameraEquipped { get; set; }
        public string PhysicalRemote { get; set; }
        public string Note { get; set; }
        public bool IsValid { get; set; }
    }
}
